tox.ini

```diff
@@ -1,21 +1,27 @@
 [tox]
-envlist = py27, py35, py36
+envlist = py27, py38
 skip_missing_interpreters = true
-requires =
-    pip >= 20.1.1
-    virtualenv >= 20.0.21
 
 [testenv]
 whitelist_externals =
-    /bin/bash
-    /usr/bin/wget
-    /usr/local/bin/wget
-    /usr/bin/java
+    bash
+    java
+    rm
+    wget
+
 deps =
     -rrequirements.txt
     -rdev-requirements.txt
-commands =
-    wget -N {env:SWAGGER_URL:https://oss.sonatype.org/content/repositories/releases/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar}
```

Comment:
Could you keep the ability to override `SWAGGER_URL` here? `docker-compose-ci.yaml` uses that to fetch the jar from Nexus instead of going out to Maven Central every time.

+1 for renaming the downloaded file so the rest of the test automation doesn't expect a particular version.

Reply:
I guess at least make it possible to replace `https://repo1.maven.org/maven2` so we can use a different Maven repository.